README.md

```diff
@@ -1,14 +1,23 @@
 # Grid Data Models (GDM)
 
-GDM is a python package containing [pydantic](https://docs.pydantic.dev/latest/) data models for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/).
+GDM is a python package containing [pydantic](https://docs.pydantic.dev/latest/) for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/) and intended to be open sourced in the future.
```

Comment:
This sentence seems awkward. Are you intending something like this?
```suggestion
GDM is a python package containing data models for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/).
```
Note that I removed the open source language.

Reply:
Thanks for the edit.